shared/structures/src/OrganizationLvlRecordsConfiguration.ts

```diff
@@ -0,0 +1,9 @@
+// todo: change name, but: confusion with OrganizationRecordsConfiguration
+
+import { ArrayDecoder, AutoEncoder, Decoder, field } from '@simonbackx/simple-encoding';
+import { RecordCategory } from './members/records/RecordCategory';
+
+export class OrganizationLvlRecordsConfiguration extends AutoEncoder {
```

Comment:
Er is nergens een patroon om afkortingen te gebruiken in class names of property names, dus zou het zeker voluit laten.


```suggestion
export class OrganizationLevelRecordsConfiguration extends AutoEncoder {
```